test_helpers/inc/c_util_test_helpers/c_util_test_env_params.h

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#ifndef C_UTIL_TEST_ENV_PARAMS_H

```

Comment:
>C_UTIL_ [](http://example.com/codeflow?start=8&length=7)

Do we need the c_util prefix? A little bit of a mouth full. Is it because we plan to have another one somewhere else? Maybe we should not have another one as it is getting a bit too much to have multiple env params for test purposes. Will get pretty confusing me thinks.

Reply:
this function was pulled out of test_env_params.h in the EBS test_utils code.  Renaming to avoid naming conflict seemed prudent.

Reply:
changing the code to accept a root path. Then we don't need this file at all and you and Matt are both happy.